backend/plugin/query.go

```diff
@@ -0,0 +1,56 @@
+package plugin
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+type Query struct {
+	RawQuery       bool
+	Target         string
+	Keyspace       string
+	Table          string
+	ColumnValue    string
+	ColumnID       string
+	ValueID        string
+	AliasID        string
+	ColumnTime     string
+	TimeFrom       time.Time
+	TimeTo         time.Time
+	AllowFiltering bool
+	Instant        bool
```

Comment:
Instant?

Reply:
It is term from prometheus datasource, that parameter has similar meaning(but not exactly due to difference between databases). https://grafana.com/docs/grafana/latest/datasources/prometheus/query-editor/#type
I used it to add some familiarity because prometheus is the most popular timeseries storage, and also I could not come up with more relevant name:)